webhooks/models.py

```diff
@@ -0,0 +1,170 @@
+from typing import Any, Optional
+from typing_extensions import Literal
+from pydantic import BaseModel, ValidationError
+
+
+class HookBase(BaseModel):
+    wallet_id: Optional[str]
+
+
+class ConnectionsHook(HookBase):
```

Comment:
This doesn't follow the examples from the webhooks issue: https://github.com/didx-xyz/aries-cloudapi-python/issues/155

What is the reason you went with the full payloads? 

The webhook models are currently not matching with the models returned from the CloudAPI. I think it would be good to keep those as closely aligned as possible as now the user needs to handle two structures for the same data payload. 

Maybe we can keep the raw webhooks in the webhooks app so we can consume the full webhook in the CloudApi, but we transform it to the CloudApi models before sending it?

Reply:
Yes, I took the liberty to go for the full ones because I thought this makes them more versatile and people can use them for other cases/don't have to extend them. I'm happy to make them as per issue though if you think that's and overkill. 

I'm all in favour of making the models as then ones returned from cloudapi though (as a personal preference of what I interpret as 3 options to go from this comment)